2023-03-14 10:21:48,171 ERROR 'Player' object has no attribute 'image'
Traceback (most recent call last):
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\game.py", line 8, in <module>
    game_loop = GameLoop()
                ^^^^^^^^^^
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\game_loop.py", line 27, in __init__
    self.setup()
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\game_loop.py", line 37, in setup
    self._01_level = _01_Main(event_loop=self.event_loop, settings=self.settings, locator=self.locator)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_01_level\_01_main.py", line 79, in __init__
    self.setup()
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_01_level\_01_main.py", line 100, in setup
    self.player = Player(
                  ^^^^^^^
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_01_level\player.py", line 8, in __init__
    super().__init__(groups, pos, collision_sprites, death_zones, settings, map_width, distance_between_rects_method)
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_01_level\entity.py", line 21, in __init__
    self.set_hitbox(pos)
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_01_level\player.py", line 109, in set_hitbox
    self.rect = self.image.get_rect(topleft=pos)
                ^^^^^^^^^^
AttributeError: 'Player' object has no attribute 'image'
